packages/kit/src/runtime/client/client.js

```diff
@@ -1389,10 +1389,17 @@ export function create_client({ target, base }) {
 				const a = find_anchor(/** @type {Element} */ (event.composedPath()[0]), container);
 				if (!a) return;
 
-				const { url, external, has } = get_link_info(a, base);
-				const options = get_router_options(a);
+				const { url, external, target } = get_link_info(a, base);
 				if (!url) return;
 
+				// bail out before `beforeNavigate` if link opens in a different tab
+				if (target === '_parent' || target === '_top') {
+					if (window.parent !== window) return;
+				} else if (target === '_blank' || target?.startsWith('_')) {
```

Comment:
Are underscored names reserved or could someone do `_google` (in which case the logic would need to be adjusted)?

Reply:
ah you're right, it should probably just be this

```suggestion
				} else if (target && target !== '_self') {
```